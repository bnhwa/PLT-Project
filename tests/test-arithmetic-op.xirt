num main() {
    num k;

    k = 1 + 9;
    printn(k);

    k = 1 + -9;
    printn(k);

    k = 9 + -4;
    printn(k);

    k = -8 + 2;
    printn(k);

     k = 9 - 4;
    printn(k);

    k = 0 - 0;
    printn(k);

    k = 3 - 4;
    printn(k);

    k = 2 - 2;
    printn(k);

    k = 8 * 2;
    printn(k);

    k = -8 * 2;
    printn(k);

     k = 8.3 * 2;
    printn(k);

    k = 2 * 0;
    printn(k);


    k = -1.3 * 0.9;
    printn(k);

    k = 1.3 * -0.9;
    printn(k);

    k = 1 / 1;
    printn(k);

    k = 1 / 0;
    printn(k);

    k = 1/ 7;
    printn(k);

    k = -1 / -7;
    printn(k);
}
