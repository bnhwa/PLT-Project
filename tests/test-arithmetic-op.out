10
-8
5
-6
5
0
-1
0
16
-16
16.6
0
-1.17
-1.17
1
inf
0.142857
0.142857